### PR TITLE
Convert to NWU format (matching ROS motion) instead of ENU

### PR DESCRIPTION
**Pull request by bhomberg**

### CODE REVIEW

**Comment by bhomberg** [comment]
If we want to upstream this, I'd probably need to add tf_ned_to_nwu as a new parameter rather than replacing tf_ned_to_enu, but I'm not sure of the priority of that.  Thoughts @jonbinney ?

**Comment by bhomberg** [comment]
@jonbinney ready for review.  Also, turns out we do have a magnetometer.  0 on the magnetometer does not seem to be quite north (roughly 30 degrees off?) at least on my desk.  I'll see if I can get it better post-calibration.

File: src/main.cpp [hunk 1]
```diff
@@ -223,40 +228,66 @@ void publishMsgs(um7::Registers& r, ros::NodeHandle* imu_nh, sensor_msgs::Imu& i
...
-      imu_msg.orientation.z = r.quat.get_scaled(3);
-
-      imu_msg.angular_velocity.x = r.gyro.get_scaled(0);
-      imu_msg.angular_velocity.y = r.gyro.get_scaled(1);
-      imu_msg.angular_velocity.z = r.gyro.get_scaled(2);
-
-      imu_msg.linear_acceleration.x = r.accel.get_scaled(0);
-      imu_msg.linear_acceleration.y = r.accel.get_scaled(1);
-      imu_msg.linear_acceleration.z = r.accel.get_scaled(2);
+        case ENU:
```
Comment (jonbinney) [inline src/main.cpp]: @bhomberg it looks like the indentation was 2 spaces and you've changed it to 4 in some places. Could you change it back? That should make it easier for me to see what has actually changed.

Comment (bhomberg) [reply src/main.cpp]: @jonbinney fixed. Ready for a review.

**Comment by bhomberg** [comment]
@jonbinney addressed your comment.  Ready for review.

**Review by jonbinney — APPROVED** [review]
LGTM!

**Pull request merged**